README.md

```diff
@@ -42,6 +42,9 @@ Build Status
 [linux-arm-build-badge]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/8721/badge
 [linux-arm-build]: https://devdiv.visualstudio.com/DevDiv/_build?_a=completed&definitionId=8721
 
+[linux-arm64-build-badge]: https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/8896/badge
+[linux-arm64-build]: https://devdiv.visualstudio.com/DevDiv/_build?_a=completed&definitionId=8896
```

Comment:
Not sure what the number at the end was so I assigned to the PR #

Reply:
@sdmaclea
I set-up the VSO build definition with triggers 'off'
definitionId=8817

Reply:
Done